### PR TITLE
Move DID method security & privacy into normative section

### PR DESCRIPTION
**Pull request by rhiaro**
Identical changes to https://github.com/w3c/did-core/pull/112 but with a clean history.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/130.html" title="Last updated on Jan 21, 2020, 2:49 PM UTC (9fabb41)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/130/e479550...9fabb41.html" title="Last updated on Jan 21, 2020, 2:49 PM UTC (9fabb41)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2275,68 +2213,146 @@ <h2>
...
 man-in-the-middle. Potential denial of service attacks MUST be
-identified as well. If the protocol incorporates cryptographic
-protection mechanisms, it should be clearly indicated which portions
+identified as well. 
+      </p>
+
+      <p>
+If the protocol incorporates cryptographic protection mechanisms,
+it should be clearly indicated which portions
 of the data are protected and what the protections are (i.e.,
```
Comment (TallTed) [inline index.html]:
```suggestion
of the data are protected and what the protections are (e.g.,
```

File: index.html [hunk 2]
```diff
@@ -2275,68 +2213,146 @@ <h2>
...
+
+  <section class='normative'>
+    <h1>
+DID Resolvers
+    </h1>
+
+    <p>
+A <a>DID resolver</a> is a software or hardware component with an API for resolving
+<a>DIDs</a> of at least one <a>DID method</a>. It
+executes the corresponding <a>DID method</a>'s <a href="#read-verify">Read</a> operation to
```
Comment (TallTed) [inline index.html]: There's another PR that edits this paragraph... The other wording is preferred, when the conflict comes to resolution.

Comment (rhiaro) [reply index.html]: I didn't edit this one, it just moved because of the rearranging.

**Comment by msporny** [comment]
Merge conflicts.

**Comment by rhiaro** [comment]
Stuck until #133 is merged

**Comment by msporny** [comment]
> Stuck until #133 is merged

#133 has been merged, please rebase @rhiaro.

**Comment by rhiaro** [comment]
Rebased, @msporny ready to merge

**Comment by msporny** [comment]
Editorial, multiple reviews, comments addressed, merging.

**Pull request merged**